### ISSUE TITLE
Installing on windows fails

### ISSUE THREAD

**Issue by Inalegwu** [YuJianrong/node-unrar.js#324]
I've installed this package in my project before but when trying to move to a new package manager and reinstall my packages , I get an error with the following

OS : Windows 11 22H2
Node version : 21.1.0
PNPM Version: 8.10.3

```
node_modules/node-unrar-js postinstall$ wget https://www.rarlab.com/rar/unrarsrc-6.1.7.tar.gz && tar -xf unrarsrc-6.1.7.tar.gz -C …
│ --2023-11-22 03:50:22--  https://www.rarlab.com/rar/unrarsrc-6.1.7.tar.gz
│ Resolving www.rarlab.com (www.rarlab.com)... 51.195.68.162
│ Connecting to www.rarlab.com (www.rarlab.com)|51.195.68.162|:443... connected.
│ HTTP request sent, awaiting response... 200
│ Length: 236798 (231K) [application/x-gzip]
│ Saving to: 'unrarsrc-6.1.7.tar.gz'
│      0K .......... .......... .......... .......... .......... 21%  161K 1s
│     50K .......... .......... .......... .......... .......... 43%  251K 1s
│    100K .......... .......... .......... .......... .......... 64%  227K 0s
│    150K .......... .......... .......... .......... .......... 86%  305K 0s
│    200K .......... .......... .......... .                    100%  433K=1.0s
│ 2023-11-22 03:50:25 (239 KB/s) - 'unrarsrc-6.1.7.tar.gz' saved [236798/236798]
│ tar: could not chdir to 'src/cpp'
```

This error hadn't happened before

**Comment by IchabodCrow** [comment]
I have the same problem on MacOS 14.1.1 (23B81)

**Comment by IchabodCrow** [comment]
@Inalegwu just install v2.0.0

**Comment by pradeep-kr27** [comment]
cpu-features@0.0.2 install /app/prj/node_modules/cpu-features
> node-gyp rebuild

make: Entering directory '/app/prj/node_modules/cpu-features/build'
  ACTION Configuring dependencies /app/prj/node_modules/cpu-features/deps/cpu_features/build/Makefile
/bin/sh: 1: cmake: not found
make: *** [config_deps.target.mk:13: /app/prj/node_modules/cpu-features/deps/cpu_features/build/Makefile] Error 127
make: Leaving directory '/app/prj/node_modules/cpu-features/build'
gyp ERR! build error 
gyp ERR! stack Error: `make` failed with exit code: 2
gyp ERR! stack     at ChildProcess.onExit (/usr/local/lib/node_modules/npm/node_modules/node-gyp/lib/build.js:194:23)
gyp ERR! stack     at ChildProcess.emit (events.js:400:28)
gyp ERR! stack     at Process.ChildProcess._handle.onexit (internal/child_process.js:285:12)
gyp ERR! System Linux 5.10.102.1-microsoft-standard-WSL2
gyp ERR! command "/usr/local/bin/node" "/usr/local/lib/node_modules/npm/node_modules/node-gyp/bin/node-gyp.js" "rebuild"
gyp ERR! cwd /app/prj/node_modules/cpu-features
gyp ERR! node -v v14.21.3
gyp ERR! node-gyp -v v5.1.1
gyp ERR! not ok 

> node-unrar-js@2.0.1 postinstall /app/prj/node_modules/node-unrar-js
> wget https://www.rarlab.com/rar/unrarsrc-6.1.7.tar.gz && tar -xf unrarsrc-6.1.7.tar.gz -C src/cpp && rm -rf unrarsrc-6.1.7.tar.gz

--2023-11-24 08:52:57--  https://www.rarlab.com/rar/unrarsrc-6.1.7.tar.gz
Resolving www.rarlab.com (www.rarlab.com)... 51.195.68.162
Connecting to www.rarlab.com (www.rarlab.com)|51.195.68.162|:443... connected.
HTTP request sent, awaiting response... 200 
Length: 236798 (231K) [application/x-gzip]
Saving to: ‘unrarsrc-6.1.7.tar.gz’

unrarsrc-6.1.7.tar.gz               100%[==================================================================>] 231.25K  83.5KB/s    in 2.8s    

2023-11-24 08:53:03 (83.5 KB/s) - ‘unrarsrc-6.1.7.tar.gz’ saved [236798/236798]

tar: src/cpp: Cannot open: No such file or directory
tar: Error is not recoverable: exiting now

npm ERR! code ELIFECYCLE
npm ERR! errno 2
npm ERR! node-unrar-js@2.0.1 postinstall: `wget https://www.rarlab.com/rar/unrarsrc-6.1.7.tar.gz && tar -xf unrarsrc-6.1.7.tar.gz -C src/cpp && rm -rf unrarsrc-6.1.7.tar.gz`
npm ERR! Exit status 2
npm ERR! 
npm ERR! Failed at the node-unrar-js@2.0.1 postinstall script.
npm ERR! This is probably not a problem with npm. There is likely additional logging output above.

npm ERR! A complete log of this run can be found in:
npm ERR!     /root/.npm/_logs/2023-11-24T08_53_06_961Z-debug.log


I'm having the same issue, so I installed node-unrar-js@2.0.0 and it's installed

**Comment by pontusvision** [comment]
same issue on ubuntu.  It looks like the tar command is failing because the src/cpp folder doesn't exist.

**Comment by Inalegwu** [comment]
> @Inalegwu just install v2.0.0

That still fails with the same error

**Comment by emmercm** [comment]
Same issue on a MacBook Air M2 on macOS Sonoma v14.1. Node.js v20.10.0, npm v10.2.3.

**Comment by Vance-ng-vn** [comment]
same error on linux

**Comment by YuJianrong** [comment]
Sorry guys, it's a bug introduced in `v2.0.1`, it's fixed now, please upgrade to `v2.0.2`.
